src/mpc/cli/mpc_phase2_begin.cpp

```diff
@@ -0,0 +1,116 @@
+#include "mpc_common.hpp"
+#include "snarks/groth16/mpc/mpc_utils.hpp"
+#include "snarks/groth16/mpc/phase2.hpp"
+
+#include <boost/program_options.hpp>
+#include <fstream>
+
+using namespace libzeth;
+namespace po = boost::program_options;
+
+namespace
+{
+
+// Usage:
+//   $0 phase2-begin [<options>] <linear_combination_file>
+//
+// Options:
+//   --out <file>    Initial challenge output file (mpc-challenge.bin)
+class mpc_phase2_begin : public subcommand
+{
+private:
+    std::string lin_comb_file;
+    std::string out_file;
+
+public:
+    mpc_phase2_begin() : subcommand("phase2-begin"), lin_comb_file(), out_file()
+    {
+    }
+
+private:
+    void initialize_suboptions(
+        boost::program_options::options_description &options,
```

Comment:
Why not use `po::` here?

Reply:
switched to `po::`

Reply:
Thanks. Several other files had similar patterns though, have you changed all occurrences of that?

Reply:
Just checked and similar changes may need to be applied to other files. Just added other comments below to flag the locations where `po::` can be used.